IDE/XCODE-FIPSv2/macOS-C++/M1/user_settings.h

```diff
@@ -0,0 +1,685 @@
+/* user_settings.h
+ *
+ * Copyright (C) 2006-2022 wolfSSL Inc.  All rights reserved.
+ *
+ * This file is part of wolfSSL.
+ *
+ * Contact licensing@wolfssl.com with any questions or comments.
+ *
+ * https://www.wolfssl.com
+ */
+
+/* Custom wolfSSL user settings for GCC ARM */
```

Comment:
Same comment as earlier. Going to assume my other comments apply to this file, too, as it seems largely the same (identical?) to the other user_settings.h.

Reply:
Will address in the M1 file also. Thanks!